plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -281,6 +298,38 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->resource = flb_sds_create(FLB_SDS_RESOURCE_TYPE);
     }
 
+    flb_kv_init(&ctx->labels);
+
+    /*
+     * Load monitored resource labels from config files.
+     * Will only use labels specified for a particular resource type at
+     * https://cloud.google.com/logging/docs/api/v2/resource-list
+    */
+    mk_list_foreach(labels, &ins->properties) {
+        kv = mk_list_entry(labels, struct flb_kv, _head);
+        if (strncasecmp(kv->key, "label.", 6) == 0 &&
```

Comment:
That sounds reasonable, will update.

Reply:
Once you update, you'd need to edit your PR comment with the example config, as well as change this `6` everywhere. Regarding the latter, it might be more robust to factor out the prefix into a static constant, e.g.,:
```c
static const char resource_label_prefix[] = "label.";
static const size_t resource_label_prefix_len = sizeof(resource_label_prefix)-1;
```

Reply:
Done